### PR TITLE
[WIP] Consider comments in EqualsVisitor as well.

### PR DESCRIPTION
**Pull request by retuxx**
Do I have to copy the file EqualsVisitor.java from target/generated/spoon/support/visitor/equals/EqualsVisitor.java to src/main/java/spoon/support/visitor/equals/EqualsVisitor.java manually? The test case CtGenerationTest.testGenerateEqualsVisitor fails because the generated and commit files differ.

### CODE REVIEW

File: src/main/java/spoon/support/visitor/equals/EqualsChecker.java [hunk 1]
```diff
@@ -254,4 +255,33 @@ public void visitCtContinue(CtContinue e) {
 		}
 		super.visitCtLiteral(e);
 	}
+
+	@Override
+	public void visitCtComment(CtComment e) {
```
Comment (tdurieux) [inline src/main/java/spoon/support/visitor/equals/EqualsChecker.java]: There is an implemented equals in CtComment, why do you not use it?

Comment (retuxx) [inline src/main/java/spoon/support/visitor/equals/EqualsChecker.java]: I don't see any equals method in [CtComment](https://github.com/INRIA/spoon/blob/master/src/main/java/spoon/reflect/code/CtComment.java)

Comment (tdurieux) [inline src/main/java/spoon/support/visitor/equals/EqualsChecker.java]: It is available in the implementation: https://github.com/INRIA/spoon/blob/master/src/main/java/spoon/support/reflect/code/CtCommentImpl.java#L61

Comment (retuxx) [inline src/main/java/spoon/support/visitor/equals/EqualsChecker.java]: But `visitCtComment` takes a `CtComment`. Should I really depend on the `CtCommentImpl` implementation?

**Pull request closed without merging**